openquake/hazardlib/gsim/abrahamson_2014.py

```diff
@@ -295,10 +296,12 @@ def _get_soil_depth_term(self, C, z1pt0, vs30):
         """
         # Get reference z1pt0
         z1ref = self._get_z1pt0ref(vs30)
+        # Get z1pt0
+        z10 = copy.deepcopy(z1pt0)
         # This is used for the calculation of the motion on reference rock
         idx = z1pt0 < 0
-        z1pt0[idx] = z1ref[idx]
-        factor = np.log((z1pt0+0.01)/(z1ref+0.01))
+        z10[idx] = z1ref[idx]
+        factor = np.log((z10+0.01)/(z1ref+0.01))
```

Comment:
Regarding PEP8 you're right Michele. I'm surprised I didn't get any message when I run pep8. 

Regarding the testing I confess I was quite surprised by this error since the method I fixed is used every time you compute ground motion with this GMPE. Therefore all the tests were using it and were all passing. However when you use the GMPE in a regular analysis an error was generated:
z1pt0[idx] = z1ref[idx]
ValueError: An error occurred with source id=3-42198. Error: assignment destination is read-only
The fix I'm adding here removes this problem.